go/go_grpc_library.bzl

```diff
@@ -1,31 +1,36 @@
+load("//go:go_proto_library.bzl", "PROTO_DEPS")
 load("//go:go_grpc_compile.bzl", "go_grpc_compile")
+load("//go:go_proto_compile.bzl", "go_proto_compile")
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
 def go_grpc_library(**kwargs):
     # Compile protos
     name_pb = kwargs.get("name") + "_pb"
-    go_grpc_compile(
+    go_proto_compile(
         name = name_pb,
         prefix_path = kwargs.get("importpath", ""),
         **{k: v for (k, v) in kwargs.items() if k in ("deps", "verbose")} # Forward args
     )
 
+    grpc_name_pb = kwargs.get("name") + "_grpc_pb"
+    go_grpc_compile(
+        name = grpc_name_pb,
+        prefix_path = kwargs.get("importpath", ""),
+        **{k: v for (k, v) in kwargs.items() if k in ("deps", "verbose")} # Forward args
+    )
+
     # Create go library
     go_library(
         name = kwargs.get("name"),
-        srcs = [name_pb],
-        deps = kwargs.get("go_deps", []) + GRPC_DEPS,
+        srcs = [name_pb, grpc_name_pb],
+        deps = kwargs.get("go_deps", []) + GRPC_DEPS + PROTO_DEPS,
         importpath = kwargs.get("importpath"),
         visibility = kwargs.get("visibility"),
         tags = kwargs.get("tags"),
     )
 
 GRPC_DEPS = [
-    "@com_github_golang_protobuf//proto:go_default_library",
-    "@org_golang_google_protobuf//reflect/protoreflect:go_default_library",
-    "@org_golang_google_protobuf//runtime/protoimpl:go_default_library",
     "@org_golang_google_grpc//:go_default_library",
     "@org_golang_google_grpc//codes:go_default_library",
     "@org_golang_google_grpc//status:go_default_library",
-    "@org_golang_x_net//context:go_default_library",
```

Comment:
This is not longer needed as gRPC now uses the standard library `context` package, which is the successor to this one.